style.css

```diff
@@ -1 +1,199 @@
+  
+  body {
+    font-family: Courgette;
+    margin:0;
+    color: #50210d;
+    padding: 1px;
+    background: #F5C7DE;
+    height: auto;
+  }
+  
+  p{
+      padding: 0px 5px;
+  }
+
+  /* Header */
+  .header {
+    padding: 10px;
+    text-align: center;
+    
+  }
+  
+  .header h1 {
+    font-size: 2rem;
+    color:#825947;
+  }
+
+  h2 {
+      color:#b81c4c
+  }
+
+  main{
+      max-width: 900px;
+      margin:auto;
+      box-shadow: 30px 0px 40px rgba(0,0,0,0.4), -30px, 0px, 40px rgba(0,0,0,0.4); /*not working*/
+  }
+  /* Style the top navigation bar */
+  .topnav {
+    overflow: hidden;
+    margin-top: 10;
+    background-color: #eb9de0;
+  }
+  
+  /* Style the topnav links */
+  .topnav a {
+    display: block;
+    background-color: rgb(233, 159, 196);
+    color: #850a2f;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    float: none;
+    width: 100%;
+    
+  }
+  
+  /* Change color on hover */
+  .topnav a:hover {
+    background-color: #b81c4c;
+    color: white;
+  }
+  
+  /* Create two equal columns that floats next to each other */
+  .leftcolumn, .rightcolumn {   
+    width: 100%;
+    padding: 0;
+  }
+
+  
+  /* image */
+  .logo {
+    width: 100%;
+    float: left;
+    }
+
+    
+    .container{
+        width:100%;
+    }
+  .img {
+    background-color: #825947;
+    width: 100%;
+    padding: 5px;
+    display:block;
+  }
+  
+  /* Add a card effect */
+  .card {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    padding: 10px;
+  }
+  
+
+   /* Create 4 equal columns that floats next to each other */ 
+   .bottom-img {
+    
+    background-color: #825947;
+    width: 100%;
+    padding: 2px;
+
+   }
+
+  /* Footer */
+  .footer {
+
+    width: 100%;
+    float: left;
+    clear: both;
+    text-align: center;
+    padding: 1em 0;
+    color: #541800;
+    background: #fff;
+    
+
+  }
+  
+  /* Responsive layout - when the screen is more than 768px wide, make the two columns stack on top of each other instead of next to each other */
+  @media screen and (min-width: 800px) {
+   
+    .row{
+        display: flex;
+        width: 100%;
+        flex-direction: row-reverse;
+        justify-content: space-between;
+    }
+  /* header */
+main{
+    max-width: 1024px;
+    margin:auto;
+}
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+    text-align: right;
+    background-image: url(/images/ValentineFirstImage.png);
+    background-size:300px;
+    background-repeat: no-repeat;
+    background-position: 50% 80%;
+}
+.container {
+    float: left;
+    width: 24%;
+}
+.container-right {
+    display: flex;
+    flex-direction: column;
+    float: right;
+    width: 30%;
+}
+
+  /* main */
+    .card {
+        background-color: white;
+        padding: auto;
+        margin-top: 10px;
+      }
+
+    .rightcolumn {   
+        width: 70%;
+        background-color: white;
+        align-self: 2;
```

Comment:
have a look at what the valid values are for [align-self](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self) - at the moment the css is invalid and would not apply any alignment to this class